### PR TITLE
Update `FabricGateway` CRD

### PR DESCRIPTION
**Pull request by MustafaSaber**
Signed-off-by: Mustafa Abdelrahman <mustafa.abdelrahman@zalando.de>

### CODE REVIEW

File: cluster/manifests/fabric-gateway/fabricgateway_crd.yaml [hunk 1]
```diff
@@ -369,5 +375,5 @@ status:
     plural: ""
   conditions: []
   storedVersions: []
-
+k
```
Comment (AlexanderYastrebov) [inline cluster/manifests/fabric-gateway/fabricgateway_crd.yaml]: ?

Comment (MustafaSaber) [reply cluster/manifests/fabric-gateway/fabricgateway_crd.yaml]: Nice catch!

Comment (mikkeloscar) [reply cluster/manifests/fabric-gateway/fabricgateway_crd.yaml]: Goof

**Comment by AlexanderYastrebov** [comment]
:+1:

**Comment by RomanZavodskikh** [comment]
:+1:

**Pull request merged**